pytket/tests/strategies.py

```diff
@@ -174,6 +174,7 @@ def backendinfo(
     supports_fast_feedforward = draw(st.booleans())
     supports_reset = draw(st.booleans())
     supports_midcircuit_measurement = draw(st.booleans())
+    # is it possible to exclude some optyps at this point
```

Comment:
Here is the `BackendInfo` created. I have not found a way to convert this into some helpful information without using the json functions.